core/src/Platforms/iOS/SystemWebview/SystemWebUI.cs

```diff
@@ -92,28 +92,27 @@ public void DidFinish(SFSafariViewController controller)
             }
         }
 
-        private UIViewController FindCurrentViewController(UIViewController rootViewController)
+        private static UIViewController FindCurrentViewController(UIViewController CallerViewController)
```

Comment:
>FindCurrentViewController [](start = 40, length = 25)

see comment in CoreUIParent ... also is it needed to have this method 2 places? re-use?

Reply:
right..not needed in two places. I will leave in CoreUIParent. thanks